api.go

```diff
@@ -946,10 +946,10 @@ type SearchOptions struct {
 	// will be used. This option is temporary and is only exposed for testing/ tuning purposes.
 	DocumentRanksWeight float64
 
-	// EXPERIMENTAL. If true, use keyword-style scoring instead of the default scoring formula.
-	// Currently, this treats each match in a file as a term and computes an approximation to BM25.
-	// When enabled, all other scoring signals are ignored, including document ranks.
-	UseKeywordScoring bool
+	// EXPERIMENTAL. If true, use text-search style scoring instead of the default scoring formula.
+	// The scoring algorithm treats each match in a file as a term and computes an approximation to
+	// BM25. When enabled, all other scoring signals are ignored, including document ranks.
+	UseBM25Scoring bool
```

Comment:
Will this break existing callers?

Reply:
Yes, and when I update sg/sg to a new Zoekt version, I will update all callsites. In general, the feature is experimental which lets us change it more freely.

Reply:
ok cool